docs/specs/code/hardware-wallets/ledger/README.md

```diff
@@ -208,6 +208,76 @@ app/components => View base components
 
 # Iteration-2
 
+## Proposal
+Updated UI to for Ledger Nano X support using USB cable.
+
+## Related PRs
+Implementation: https://github.com/Emurgo/yoroi-frontend/pull/660
+
+# Iteration-3
+
+## Proposal
+As in Windows 10, version 1903 (10th May 2019 update) native support for U2F has been added and it's affecting current Yoroi UI.<br>
+Demo can be found here: https://youtu.be/Hlo6wOA16MA
+
+In order fix UI problem, support for [WebAuthn Transport](https://www.npmjs.com/package/@ledgerhq/hw-transport-webusb) is proposed.
+
+As WebAuthn Transport supports all [targetted browsers](https://caniuse.com/#search=Web%20Authentication%20API) it will be used as the default protocol. U2F transport will still be available.
+
+## Module structure
+Till now we have two modules and one repository at: https://github.com/Emurgo/yoroi-extension-ledger-bridge<br>
+`master` branch acts as connector module<br>
+`gh-pages` branch is the actual `iframe` which communicates with the Ledger device<br>
+making one repository per module feels more better now and it will help proper testing of new URL before release.<br>
+Hence proposing two new repository like:<br>
+1. `yoroi-extension-ledger-connect-handler` as handler of opening, closing and communication with `yoroi-extension-ledger-connect-website`
+2. `yoroi-extension-ledger-connect-website` as website which will communicate with the Ledger device
+
+Also proposing to change hosting from Github pages to **Netlify**, because debugging and development is bit difficult with Github pages.
+With **Netlify** we can set `mainnet` for **Production Netlify** and for other networks we can use **Development Netlify**.
+
+## Ledger Connect page UI design
+WebAuthn-Transport-URL types:<br>
+1. https://ledger-connect.yoroi-wallet.com/?language=ja-JP<br>
+**(Will be used from Yoroi)**<br>
+In worse case scenario, if **WebAuthn is disabled**  (i.e [isSupported](https://github.com/LedgerHQ/ledgerjs/blob/5456188404c66f10ad8be60a29a9935a8af1856f/packages/hw-transport-webauthn/src/TransportWebAuthn.js#L52) is `false`) we will fallback to **U2F**. And if **U2F** is also disabled then error will thrown asking to enable **WebAuthn**.
+
+2. https://ledger-connect.yoroi-wallet.com/?transport=webauthn<br>
+(No Fallback, Will not be used from Yoroi)
+3. https://ledger-connect.yoroi-wallet.com/?transport=webauthn&language=en-US<br>
+(No Fallback, Will not be used from Yoroi)
+
+U2F-Transport-URL types:<br>
+1. https://ledger-connect.yoroi-wallet.com/?transport=u2f<br>
+(No Fallback, Will not be used from Yoroi, just kept it as it's already in implementation)
+
+2. https://ledger-connect.yoroi-wallet.com/?transport=u2f&language=ja-JP<br>
+(No Fallback, Will not be used from Yoroi, just kept it as it's already in implementation)
+
```

Comment:
I would explicitly mention WebUSB as a transport  option even if we aren't using it for Yoroi

Reply:
ATM, WebUSB is not supported hence will write one line explaining that.